oklayoutinflator/src/main/java/tech/okcredit/layout_inflator/OkLayoutInflater.kt

```diff
@@ -90,4 +91,12 @@ class OkLayoutInflater(context: Context) {
             }
         }
     }
+
+
+    override fun onDestroy(owner: LifecycleOwner) {
+        super.onDestroy(owner)
+        coroutineContext.cancel()
```

Comment:
The original code didn't have a `cancel()` call. Any reason for adding it?

Reply:
Parent and children job should be canceled to avoid memory leaks.